examples/custom-field-react-starter/DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,169 @@
+# 🚀 Developer Guide: Creating a Custom Field with MyWave UI
+
+## 1. Clone the Starter Template
+Begin by cloning the custom field starter template:
+
+```
+git clone https://github.com/mywave-me/mywave-ui-examples.git
+cd mywave-ui-examples
+```
+
+### Prerequisites
+Ensure you have the following installed:
+
+- Node.js (v20.12.2)
+- pnpm (v9.0.6)
+
+If not, run the setup script:
+
+```
+./scripts/preinstall.sh
+```
```

Comment:
This is more root monorepo stuff described in the root `README.md`. Does it really need to be here?